nuggetsapp/urls.py

```diff
@@ -1,7 +1,10 @@
-from django.conf.urls import url
+from django.conf.urls import url, include
 from . import views
 
 urlpatterns = [
+    url(r'^api/v1/nuggets/(?P<user_id>[0-9]+)/$', views.nuggets_for_user, name='nuggets_for_user'),
```

Comment:
I like api/v0/user/123/nuggets. I'll use that

I still need to learn more about auth and make these calls require auth tokens, so that will come later

I can use a similar format to create/add/update/delete nugget, I just wanted to get eyes on the general direction of things